docs/components/store.md

```diff
@@ -136,6 +136,12 @@ Thanos Store Gateway might not get new blocks immediately, as Time partitioning
 
 We recommend having overlapping time ranges with Thanos Sidecar and other Thanos Store gateways as this will improve your resiliency to failures.
 
-Thanos Querier deals with overlapping time series by merging them together. 
+Thanos Querier deals with overlapping time series by merging them together.
 
 Filtering is done on a Chunk level, so Thanos Store might still return Samples which are outside of `--min-time` & `--max-time`.
+
+## Probes
+
+- Thanos Store exposes two endpoints for probing.
```

Comment:
Can we add: 

`NOTE: Metric endpoint starts immediately so, make sure you set up readiness probe on designated HTTP `/-/ready` path.`